tests/scripts/all.sh

```diff
@@ -933,8 +929,7 @@ component_test_ecp_no_internal_rng () {
     scripts/config.py unset MBEDTLS_CTR_DRBG_C
     scripts/config.py unset MBEDTLS_HMAC_DRBG_C
     scripts/config.py unset MBEDTLS_ECDSA_DETERMINISTIC # requires HMAC_DRBG
-    scripts/config.py unset MBEDTLS_PSA_CRYPTO_C # requires a DRBG
-    scripts/config.py unset MBEDTLS_PSA_CRYPTO_STORAGE_C # requires PSA Crypto
+    scripts/config.py set MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
```

Comment:
On second thoughts, I don't think it's a good idea to combine the testing of `MBEDTLS_ECP_NO_INTERNAL_RNG` with the testing of `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG`: I'm working on a follow-up which allows the ECP module to use `MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG`. So I'm going to amend this commit. I'll wait until no one has pending review comments to rebase.